buildscript/generate-package-infos.gradle

```diff
@@ -0,0 +1,27 @@
+import java.util.regex.Matcher
+
+def generatePackageInfos = tasks.register('generatePackageInfos', Task) {
```

Comment:
This could be in NG. (Probably should be)

Reply:
tasks for a later time™